app/models/miq_alert_status_action.rb

```diff
@@ -0,0 +1,30 @@
+class MiqAlertStatusAction < ApplicationRecord
```

Comment:
I'd be interested in having that enforced, did not find a way to do that. @kbrock do we have a mechanism for read only records? 
They should only be created & deleted but never updated